Makefile

```diff
@@ -158,9 +158,27 @@ update-contextual-logging: $(LOGCHECK)
 	UPDATE=true ./hack/verify-contextual-logging.sh
 .PHONY: update-contextual-logging
 
-generate-docs:
-	go run hack/generate/cli-doc/gen-cli-doc.go
-	./hack/generate/crd-ref/run-crd-ref-gen.sh
+kcp-docs-image: docs/Dockerfile
+	# Skip if LOCAL is set
+	[[ -n "$$LOCAL" ]] || docker buildx build -f docs/Dockerfile -t kcp-docs --load docs
```

Comment:
Why the local option? For quicker iteration?

Reply:
Non containerized